.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 *.log
 */linuxDash.min.js.map
 temp
+package-lock.json
```

Comment:
Why add the package lock to gitignore?

Reply:
When I cloned this project for the first time, `package-lock.json` did not exist (btw. This file appeared when I did npm packaging). Although I  think this file should exist, but I'm not sure why it is not available in the current repository. So in order to avoid other problems, I added this line to the ignore file.

Reply:
@afaqurk  @Arteneko   If you feel that adding `package-lock.json` is inappropriate, I can remove it in PR